src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-multisite-zone-deletion-form/rgw-multisite-zone-deletion-form.component.ts

```diff
@@ -12,23 +13,31 @@ import { NotificationService } from '~/app/shared/services/notification.service'
   templateUrl: './rgw-multisite-zone-deletion-form.component.html',
   styleUrls: ['./rgw-multisite-zone-deletion-form.component.scss']
 })
-export class RgwMultisiteZoneDeletionFormComponent implements OnInit {
+export class RgwMultisiteZoneDeletionFormComponent implements OnInit, AfterViewInit {
   zoneData$: any;
+  poolList$: any;
   zone: any;
   zoneForm: CdFormGroup;
   displayText: boolean = false;
+  includedPools: Array<string> = [];
```

Comment:
```suggestion
  includedPools: Set<string> = [];
```
If you use a `Set` you will make your code easier to follow as it will remove unnecessary `this.includedPools.includes(something)` before adding something as it implicitly bookeeps what values are inserted.

```
> x = new Set
Set(0) {}
> x.add(1).add(1)
Set(1) { 1 } // no extra book keeping was necessary
```